src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -354,6 +360,13 @@ HSHomeObject::blob_put_get_blk_alloc_hints(sisl::blob const& header, cintrusive<
         return folly::makeUnexpected(homestore::ReplServiceError::FAILED);
     }
 
+    auto pg_iter = _pg_map.find(msg_header->pg_id);
```

Comment:
NIT: can we change this part to `is_pg_present`

Reply:
Sure, thanks

Reply:
Here we need to get the pg_iter as we are going to check index, so we can't use `is_pg_present` directly.

Reply:
Found there is already has a `pg_exists` function, remove `is_pg_present`.